### ISSUE TITLE
Parse Grant County results

### ISSUE THREAD

**Issue by nk9** [openelections/openelections-data-or#164]
Results from Grant County. Not available yet.

**Comment by nk9** [comment]
Update from the county clerk:

> I made copies of 2000 primary, all races included, and the cost will be $11.50 for just that election.  If you want all races I will need a credit card number with approval to charge up to $100 not including the bank charge or a blank “Not to Exceed” check made out to the Grant County Clerk.   If you want only state level results it will be approximately ½ of that amount.

When asked about the photocopies, she said:

> I am sorry to say that, being such a small county, my elections prior to 2006 are all hard copy.  I would still need to copy and then scan and email.

So I guess maybe the older results are bound and she can't scan them without copying them? Dunno. Anyway, $50 for 8 elections isn't bad. When I get the word back from OR SoS, we can decide whether to pay.

**Comment by dwillis** [comment]
Yeah, I suppose the other option for pre-2006 is to find someone willing to go to the office and photograph the results. We've done that in other places, but not many as remote as Grant.
